abb_irb1200_support/urdf/irb1200_5_90_macro.xacro

```diff
@@ -139,14 +139,16 @@
       <axis xyz="0 0 1"/>
       <parent link="${prefix}base_link"/>
       <child link="${prefix}link_1"/>
-      <limit effort="0" lower="-2.967" upper="2.967" velocity="5.027"/>
+      <limit effort="10000" lower="-2.967" upper="2.967" velocity="5.027"/>
+      <dynamics damping="1.0" friction="1.0"/>
```

Comment:
Yeah, probably. Gazebo doesn't seem to be too sensitive to these values (not nearly as much as to the PID values in the controller). But they need to be non-zero so that the PID controller doesn't oscillate like crazy.

Reply:
Might be worth leaving a comment then, to be clear that we didn't determine these somehow the next time we're debugging gazebo?

Also, do you want me to merge or are you merging?